Gruntfile.js

```diff
@@ -188,6 +177,21 @@ module.exports = (grunt) => {
     }
   });
 
+  grunt.registerTask('checkPrettier', 'Verify that all files are correctly formatted.', function() {
+    const done = this.async();
+    childProcess.exec('prettier -l . bin/*', (err, stdout, stderr) => {
+      const files = stdout.trim();
+      if (files) {
+        return done(new Error(`Files with incorrect formatting (run "npm run format" and consider a Prettier plugin for your editor):\n${files}\n`))
+      }
+      if (err) {
+        console.error(stderr)
```

Comment:
Use `grunt.log` instead of console.

Also, the line is missing a `;`. 

```suggestion
        grunt.log.error(stderr);
```